tenacity/tests/test_tenacity.py

```diff
@@ -382,7 +382,7 @@ def go(self):
 
 
 class NoIOErrorAfterCount(object):
-    "Holds counter state for invoking a method several times in a row."
+    """Holds counter state for invoking a method several times in a row."""
```

Comment:
> but I don't like making style only pull requests as it simply makes git blame harder to read.

What makes git blame hard to read is the change itself, not that it is a specific PR.

But it's not big deal. I wrote "would be great" so I can pinch my nose ;)